<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>347</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Texas Holdem Settings</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>220</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <widget class="QRadioButton" name="rServer">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbVictory">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbCredit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbPlayers">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="sbBlind">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>40</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Max Players (up to 9)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>70</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string># of Players Remaing</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>100</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Starting Credit</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>130</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Big Blind Amount</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-or-</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="rClient">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>50</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Client</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>75</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbContinue">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>170</y>
            <width>75</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Continue</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>(Name have a length greather than 2.)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Create Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>999999999</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbContinue</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>rServer</tabstop>
  <tabstop>rClient</tabstop>
  <tabstop>sbPlayers</tabstop>
  <tabstop>sbVictory</tabstop>
  <tabstop>sbCredit</tabstop>
  <tabstop>sbBlind</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
